lib/prom_ex/plugins/phoenix.ex

```diff
@@ -330,12 +330,14 @@ if Code.ensure_loaded?(Phoenix) do
             reporter_options: [
               buckets: [10, 100, 500, 1_000, 5_000, 10_000]
             ],
-            tag_values: fn %{socket: %Socket{endpoint: endpoint}} ->
+            tag_values: fn %{socket: %Socket{endpoint: endpoint, handler: handler}, event: event} ->
               %{
-                endpoint: normalize_module_name(endpoint)
+                endpoint: normalize_module_name(endpoint),
+                event: event,
```

Comment:
Overall this looks good to me. The only part that gives me pause is the `event` since it is a binary and has the potential to be an unbounded value. I think it would be best if this module accepts an option to normalize event values. The default implementation would be a passthrough:

```elixir
fn event -> 
  event
end
```

But then the user could override this to do something like so:

```elixir
fn 
  "join" -> "join"
  "leave" -> "leave"
  _ -> "unknown"
end
```

Thoughts?

Reply:
Hey @akoutmos good idea and I can implement it like you've described. Just might be a little slow due to the holiday season!

Reply:
No worries at all my friend! Happy holidays!

Reply:
I've updated the PR with support for a normalizer function. I also added some new tests- one to test for `handled_in` metrics generally which I noticed were missing and another to test that configuring the `normalize_event_names` option works as expected.